lms/templates/registration/password_reset_confirm.html

```diff
@@ -1,72 +1,75 @@
-{% extends "main_django.html" %}
-{% load i18n %}
+## mako
 
-{% block title %}
-<title>
-{% blocktrans with platform_name=platform_name %}
-    Reset Your {{ platform_name }} Password
-{% endblocktrans %}
-</title>
-{% endblock %}
+<%! from django.utils.translation import ugettext as _ %>
 
-{% block bodyextra %}
-<script type="text/javascript" src="{{STATIC_URL}}js/student_account/password_reset.js"></script>
-{% endblock %}
+<%inherit file="../main.html"/>
 
-{% block bodyclass %}view-passwordreset{% endblock %}
+<%block name="title">
+    <title>${_("Reset Your {platform_name} Password".format(platform_name=platform_name))}</title>
+</%block>
 
-{% block body %}
+<%block name="bodyextra">
+    <script type="text/javascript" src="${STATIC_URL}js/student_account/password_reset.js"></script>
+</%block>
+
+<%block name="bodyclass">view-passwordreset</%block>
+
+<%block name="body">
 <div id="password-reset-confirm-container" class="login-register">
     <section id="password-reset-confirm-anchor" class="form-type">
         <div id="password-reset-confirm-form" class="form-wrapper" aria-live="polite">
-            <div class="status submission-error {% if not err_msg %} hidden {% endif %}">
-                <h4 class="message-title">{% trans "Error Resetting Password" %}</h4>
+            <div class="status submission-error ${'hidden' if err_msg is None else ''}">
+                <h4 class="message-title">${_("Error Resetting Password")}</h4>
                 <ul class="message-copy">
-                {% if err_msg %}
-                    <li>{{err_msg}}</li>
-                {% else %}
-                    <li>{% trans "You must enter and confirm your new password." %}</li>
-                    <li>{% trans "The text in both password fields must match." %}</li>
-                {% endif %}
+                % if err_msg:
+                    <li>${err_msg}</li>
```

Comment:
Enable escaping by default to ensure that this is safe: http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/conventions/safe_templates.html